hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -1351,7 +1351,7 @@
     },
     { name:     "FW_OV_RD_FIFO_OVERFLOW",
       desc:     "Firmware override observe FIFO overflow status",
-      swaccess: "rw0c",
+      swaccess: "ro",
```

Comment:
This is not a documentation change :-)
Maybe it doesn't make sense to split up this small PR. It's only +16 lines...

Reply:
I squashed the first two commits.